### ISSUE TITLE
Uncaught TypeError: proj4 is not a function

### ISSUE THREAD

**Issue by ezekiel747** [derhuerst/transform-coordinates#3]
Hi, trying to use this package in browser and i get the following error:
`Uncaught TypeError: proj4 is not a function`

Bundling is done by ParcelJS.
Tried both with import and require(), same result.
How can i make it work?

No luck with the suggestion i found  _const proj4 = require("proj4").default_
since i'm not directly calling proj4() - just as in the example from Readme file.

Thank you in advance.

**Comment by derhuerst** [comment]
I cannot reproduce the issue. Please provide more details about your setup:

- Your `package-lock.json`/`yarn.lock` file.
- Your code calling `transform-coordinates`.
- Your Parcel config.

**Comment by ezekiel747** [comment]
Hi, i've created this sandbox, has the same error: https://codesandbox.io/s/happy-varahamihira-cfc7g6?file=/index.js

On my local dev env i had latest version of parcel, but with the same error on proj4.
I don't have the exact files from yesterday.
If needed, i can take some time to create the version i had yesterday - please let me know.
Thank you!

LE: had no custom config for parcel. Just what comes out of the box: 
npx parcel index.html

**Comment by derhuerst** [comment]
I managed to reproduce the error locally.

I think this is an issue with the way Parcel handles CommonJS and ESM code, because `transform-coordinates` & `proj4` *work*; The error happens because Parcel lets the CommonJS `require('proj4')` code in `transform-coordinates/index.js` return an ESM Module object instead of the `proj4()` function.

TLDR: It is an issue with Parcel, not with this package.

**Comment by derhuerst** [comment]
Maybe this is related? https://github.com/parcel-bundler/parcel/issues/7888

**Comment by Sheldonfrith** [comment]
Same error using nextjs, which I'm pretty sure doesn't use parcel.

**Comment by Sheldonfrith** [comment]
This solved it for me: https://stackoverflow.com/a/65379692/4205839

**Comment by derhuerst** [comment]
Sounds like this is related to how people `import`/`require()` the proj4 lib, not to `transform-coordinates`.

**Comment by Sheldonfrith** [comment]
To be clear, changing this line, in transform-coordinates: https://github.com/derhuerst/transform-coordinates/blob/be841191c1de872373a1e8f0453c876773332ebb/index.js#L4 to `const proj4 = require("proj4").default` fixed it for me.
